deploy/README.md

```diff
@@ -18,13 +18,14 @@ $ LOAD_BALANCER=route ./deploy/openshift/deploy.sh
 
```

Comment:
(not about this line)

It seems changes are applied fine in this PR (except of one case I highlighted).

I did not go to check what each separate included PR means or fixes, but the ones about PSP and CronJobs that are _included_ definitely seem relevant.

My main question: what was the method you used to determine which changes to pick for this PR?
How do we make sure that there are no other PRs that **should** be included too?

Reply:
Looks like all items returned from the following commands are here
```sh
git log -G PodSecurityPolicy
git log -G PSP
git log -G batch/v1
```